model/src/pyrenew/model/rtinfectionsrenewal.py

```diff
@@ -11,7 +12,22 @@
     ["Rt", "latent", "observed"],
     defaults=[None, None, None],
 )
-"""Output from RtInfectionsRenewalModel.sample()"""
+RtInfectionsRenewalSample.__doc__ = """
```

Comment:
Same comment as above on `namedtuple` documentation and the `typing.NamedTuple` option.